File: Chapter5/Selectionsort.cpp
#include<iostream>
using namespace std;
int main(){
    int b;
    cout<<"Enter number of elements";
    cin>>b;
    int arr[b];
    for(int i=0;i<b;i++){
        cin>>arr[i];
    }
     for(int i=0;i<b-1;i++){
         for(int j=i+1;j<b;j++){
             if(arr[j]<arr[i]){
             int temp=arr[j];
             arr[j]=arr[i];
             arr[i]=temp;
             }
         }
     }
     for(int i=0;i<b;i++){
         cout<<arr[i]<<" ";
     }
  return(0);
}

File: Chapter5/Selectionsort.py
def SelectionSort(arr, r):
    for i in range(0, r - 1):
        min1 = i
        for j in range(i + 1, r):
            if arr[min1] > arr[j]:
                min1 = j

        arr[min1], arr[i] = arr[i], arr[min1]
    return arr


arr = []
r = int(input("Enter the number of element in array"))
for i in range(0, r):
    ele = int(input(f"Enter the {i} element: "))
    arr.append(ele)

print(f"The Selection Sort is: {SelectionSort(arr, r)}")
